src/atoms/Heading.tsx

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from 'styled-components'
+
+interface HeadingProps {
+  as?: string
```

Comment:
You don't need to do that, styled components should have `as` prop automatically.

Reply:
![image](https://user-images.githubusercontent.com/33199975/70429525-05451580-1a79-11ea-98e1-99c02dcadbf8.png)
![Screenshot (698)](https://user-images.githubusercontent.com/33199975/70429545-0d9d5080-1a79-11ea-82d0-1a6b29ffade0.png)
If i do not do it , it will be an arror.

Reply:
Got it! Leave it be for now then.